src/components/growth/Growth.js

```diff
@@ -3,50 +3,73 @@ import './Growth.css';
 import solar1 from './solar_production.png';
```

Comment:
These are all changes from running `npm run lint`. I did not make any changes to these files (Growth.js or css) directly.